src/bin/cmd/client.rs

```diff
@@ -149,6 +158,27 @@ impl HTTPNodeClient {
 		e.reset().unwrap();
 	}
 
+	pub fn verify_chain(&self, fast_validation: bool) {
```

Comment:
why not use the name assume_valid_rangeproofs_kernels here as well?

Reply:
Done but i let `--fast` as the option for the arg verify-chain.